server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CommonCobolLexer.g4

```diff
@@ -15,239 +15,302 @@
 lexer grammar CommonCobolLexer;
 import TechnicalLexer;
 
+ACCESS : A C C E S S;
 ACTION : A C T I O N;
-ACTIVITY : A C T I V I T Y {!sqlFlag}?;
+ACTIVE : A C T I V E;
 ADD : A D D;
 ADDRESS : A D D R E S S;
 AFTER : A F T E R;
 ALIAS : A L I A S;
+ALL : A L L;
 ALLOCATE : A L L O C A T E;
 ALTER : A L T E R;
+ALWAYS : A L W A Y S;
 AND : A N D;
 ANY : A N Y;
+ARE : A R E;
 AS : A S;
+ASCII : A S C I I;
 AT : A T;
 ATTRIBUTES : A T T R I B U T E S;
 AUTHENTICATE : A U T H E N T I C A T E;
 AUXILIARY : A U X I L I A R Y;
+BEFORE : B E F O R E;
 BINARY : B I N A R Y;
+BIND : B I N D;
 BIT : B I T;
+BLOB : B L O B;
 BY : B Y;
+CALL : C A L L;
 CCSID : C C S I D;
 CHANGE : C H A N G E;
+CHANGED : C H A N G E D;
 CHAR : C H A R;
+CHARACTER : C H A R A C T E R;
```

Comment:
Why you move e.g. CHARACTER : C H A R A C T E R (just one example here) into common.  It is only for DB2.

Reply:
It is also used in CobolParser